link/kprobe.go

```diff
@@ -230,6 +230,11 @@ func pmuProbe(typ probeType, symbol, path string, offset uint64, ret bool) (*per
 	if errors.Is(err, os.ErrNotExist) || errors.Is(err, unix.EINVAL) {
 		return nil, fmt.Errorf("symbol '%s' not found: %w", symbol, os.ErrNotExist)
 	}
+	// Since commit cb9a19fe4aa51, ENOTSUPP is returned when attempting to set
+	// a uprobe on a trap instruction.
+	if errors.Is(err, unix.ENOTSUPP) {
+		return nil, fmt.Errorf("setting a uprobe on a trap insn (offset %#x) is not supported: %w", offset, err)
```

Comment:
Since this is probably no the only reason for ENOTSUPP, make the language less strong?
```
failed to set uprobe on offset (possible trap insn): %w
```

Reply:
It's the only occurrence in `kernel/events/`, but I'll change it. This errno has a bit of an interesting history; it's [not part of UAPI and shouldn't be propagated to userspace](https://linux.kernel.narkive.com/dkX5RyfC/question-usage-of-enotsupp-error-code), yet over the years it's made its way into a bunch of drivers, and more recently `bpf/` started using it a bunch as well.